src/main/java/com/battleship/backend/BoardRepository.java

```diff
@@ -0,0 +1,28 @@
+package com.battleship.backend;
+
+import com.battleship.backend.models.Board;
+import org.springframework.stereotype.Component;
+
+@Component
+public class BoardRepository {
+    Board[] boards;
+    Board playerBoard;
+    Board computerBoard;
+
+    public BoardRepository(){
+        playerBoard = new Board("Player Board");
+        computerBoard = new Board("Computer Board");
+        boards = new Board[] { playerBoard, computerBoard};
+    }
```

Comment:
Do you have a Java linter?  I see some spacing issues here.

Reply:
I've just been using the built in IntelliJ linter. Looking at CheckStyle now.

Reply:
Ok.  I'm surprised intellij allowed the above spacing.  Don't spend too much time on a linter.  Move on if you can't get one to work.